app/src/routes/+server.js

```diff
@@ -5,15 +5,12 @@ import { json } from '@sveltejs/kit'
 export async function GET({ locals: { db } }) {
 	// https://www.sqlite.org/lang_expr.html#the_like_glob_regexp_match_and_extract_operators
 	const sql = `
-		SELECT name
-		FROM sqlite_master
-		WHERE type = 'table'
-			AND name NOT LIKE '!_%' ESCAPE '!' -- sometimes, CF puts tables in the database, e.g., _cf_KV
-		ORDER BY name
+		SELECT DISTINCT language
+		FROM Form_Names
```

Comment:
Just curious why you chose to select from `Form_Names`? It's fine for now because we only have the English db, but it is possible that a target language does not include any unique forms, so there would be no entries for it in this db. It would probably be best to take from the the `Lexicon` table.

Reply:
✅ done in [61b96e8](https://github.com/presciencelabs/tabitha-targets/commit/61b96e80bfda25f84748932a4a83ce068c57a0c2)